website/content/docs/nia/cli/index.mdx

```diff
@@ -82,4 +82,10 @@ consul-terraform-sync task disable -port=2000 task_a
 
 Below are options that can be used across all commands:
 
-- `-port=<port>` - (int: 8558) The port that the Consul-Terraform-Sync daemon serves its API.
+- `-port=<port>` - (int: 8558) The port that the Consul-Terraform-Sync daemon serves its API. This port is used with the address `http://localhost`. If a different scheme or address is required, use `-http-addr` instead.
+- `-http-addr=<address>` - (string: http://localhost:8558) The address and port of the Consul-Terraform-Sync API. The value can be an IP address or DNS address. This can also be specified via the `CTS_HTTP_ADDR` environment variable.
+- `-ssl-verify` - (bool: true) Whether to verify TLS/SSL connections to the Consul-Terraform-Sync API. The default value is `true`, as disabling verification is a potential security vulnerability. This can also be specified using the `CTS_SSL_VERIFY` environment variable.
+- `-ca-cert=<ca_file>` - (string) The path to a PEM-encoded certificate authority file used to verify TLS/SSL connections. This can also be specified using the `CTS_CACERT` environment variable.
+- `-ca-path=<ca_directory>` - (string) The path to a directory of PEM-encoded certificate authority files used to verify TLS/SSL connections. This can also be specified using the `CTS_CAPATH` environment variable.
+- `-client-cert=<cert>` - (string) The path to a PEM-encoded client certificate used when the Consul-Terraform-Sync API has `verify_incoming` enabled. This can also be specified using the `CTS_CLIENT_CERT` environment variable.
+- `-client-key=<key>` - (string) The path to a PEM-encoded client key for the certificate specified by `-client-cert`. Used when the Consul-Terraform-Sync API has `verify_incoming` enabled. This can also be specified using the `CTS_CLIENT_KEY` environment variable.
```

Comment:
```suggestion
| Option         | Description                                                                                                                                                                                                                                                                                             | Required | Default            |
|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------|
| `-port`        | Integer value that specifies the port from which the Consul-Terraform-Sync daemon serves its API.<br/>The value is prepended with `http://localhost:`, but you can specify a different scheme or address with the `-http-addr` if necessary.                                                            | Optional | `8558`             |
| `-http-addr`   | String value that specifies the address and port of the Consul-Terraform-Sync API. <br/>You can specify an IP or DNS address.<br/> Alternatively, you can specify a value using the `CTS_HTTP_ADDR` environment variable.                                                                               | Optional | `http://locahost:` |
| `-ssl-verify`  | Boolean value that configures Consul-Terraform-Connect to enable verification for TLS/SSL connections to the API. <br/>Alternatively, you can specify the value using the `CTS_SSL_VERIFY` environment variable.                                                                                        | Optional | `true`             |
| `-ca-cert`     | String value that specifies the path to a PEM-encoded certificate authority file that is used to verify TLS/SSL connections. <br/>Alternatively, you can specify the value using the `CTS-CACERT` environment variable.                                                                                 | ?        | ?                  |
| `-ca-path`     | String value that specifies the path to a directory containing a PEM-encoded certificate authority file that is used to verify TLS/SSL connections. <br/>Alternatively, you can specify the value using the `CTS-CAPATH` environment variable.                                                          | ?        | ?                  |
| `-client-cert` | String value that specifies the path to a PEM-encoded client certificate that the Consul-Terraform-Sync API uses when the `verify_incoming` parameter is set to `true`. <br/>Alternatively,  you can specify the value using the `CTS_CLIENT_CERT` environment variable.                                | ?        | ?                  |
| `-client-key`  | String value that specifies the path to a PEM-encoded client key for the certificate configured with the `-client-cert` option. The value is used when the `verify_incoming` parameter is set to `true`. <br/>Alternatively, you can specify the value using the `CTS_CLIENT_KEY` environment variable. | ?        | ?                  |
```
It's kind of hard to see it in a GH suggestion, but I think we should start trying to transition to table formats for this kind of information. Here is the markdown for the table. I don't know what all the default values are or if any of the flags are required, though:

| Option         | Description                                                                                                                                                                                                                                                                                             | Required | Default            |
|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------|
| `-port`        | Integer value that specifies the port from which the Consul-Terraform-Sync daemon serves its API.<br/>The value is prepended with `http://localhost:`, but you can specify a different scheme or address with the `-http-addr` if necessary.                                                            | Optional | `8558`             |
| `-http-addr`   | String value that specifies the address and port of the Consul-Terraform-Sync API. <br/>You can specify an IP or DNS address.<br/> Alternatively, you can specify a value using the `CTS_HTTP_ADDR` environment variable.                                                                               | Optional | `http://locahost:` |
| `-ssl-verify`  | Boolean value that configures Consul-Terraform-Connect to enable verification for TLS/SSL connections to the API. <br/>Alternatively, you can specify the value using the `CTS_SSL_VERIFY` environment variable.                                                                                        | Optional | `true`             |
| `-ca-cert`     | String value that specifies the path to a PEM-encoded certificate authority file that is used to verify TLS/SSL connections. <br/>Alternatively, you can specify the value using the `CTS-CACERT` environment variable.                                                                                 | ?        | ?                  |
| `-ca-path`     | String value that specifies the path to a directory containing a PEM-encoded certificate authority file that is used to verify TLS/SSL connections. <br/>Alternatively, you can specify the value using the `CTS-CAPATH` environment variable.                                                          | ?        | ?                  |
| `-client-cert` | String value that specifies the path to a PEM-encoded client certificate that the Consul-Terraform-Sync API uses when the `verify_incoming` parameter is set to `true`. <br/>Alternatively,  you can specify the value using the `CTS_CLIENT_CERT` environment variable.                                | ?        | ?                  |
| `-client-key`  | String value that specifies the path to a PEM-encoded client key for the certificate configured with the `-client-cert` option. The value is used when the `verify_incoming` parameter is set to `true`. <br/>Alternatively, you can specify the value using the `CTS_CLIENT_KEY` environment variable. | ?        | ?                  |

Reply:
Makes sense, I can commit your change and edit the defaults! As a side note, do you know if there's a way to set the column width on the tables? It would be nice if the options could be on one line, but they're wrapping because of the hyphens. Not a big deal if not, just wanted to ask if it was an easy format change!

Reply:
Yeah, MD tables kind of suck, but it's what we have. The only way to widen a column is to add &nbsp; characters to a cell and force the table to adapt. We _could_ use HTML to create the tables, but the trade-off is that it breaks MD formatting, so links and back-ticks render as plaintext.

Reply:
Gotcha, I'll add the `&nbsp;` then!